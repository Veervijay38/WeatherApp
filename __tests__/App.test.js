import React from "react";
import { render, fireEvent } from "@testing-library/react-native";
import AppNavigation from "../App";
import WeatherScreen from "../src/screens/weatherScreen";

describe("AppNavigation Component", () => {
  it("renders without errors and matches snapshot", () => {
    const { toJSON } = render(<AppNavigation />);
    expect(toJSON()).toMatchSnapshot();
  });
});
