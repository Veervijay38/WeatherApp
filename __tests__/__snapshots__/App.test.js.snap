// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNavigation Component renders without errors and matches snapshot 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    {
      "flex": 1,
    }
  }
>
  <RNSScreenStack
    style={
      {
        "flex": 1,
      }
    }
  >
    <RNSScreen
      collapsable={false}
      gestureResponseDistance={
        {
          "bottom": -1,
          "end": -1,
          "start": -1,
          "top": -1,
        }
      }
      nativeBackButtonDismissalEnabled={false}
      onAppear={[Function]}
      onDisappear={[Function]}
      onDismissed={[Function]}
      onGestureCancel={[Function]}
      onHeaderBackButtonClicked={[Function]}
      onNativeDismissCancelled={[Function]}
      onTransitionProgress={[Function]}
      onWillDisappear={[Function]}
      replaceAnimation="push"
      sheetAllowedDetents="large"
      sheetCornerRadius={-1}
      sheetExpandsWhenScrolledToEdge={true}
      sheetGrabberVisible={false}
      sheetLargestUndimmedDetent="all"
      stackPresentation="push"
      style={
        {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
      swipeDirection="horizontal"
    >
      <View
        accessibilityElementsHidden={false}
        importantForAccessibility="auto"
        style={
          {
            "flex": 1,
            "flexDirection": "column-reverse",
          }
        }
      >
        <View
          collapsable={false}
          style={
            [
              {
                "flex": 1,
              },
              {
                "backgroundColor": "rgb(242, 242, 242)",
              },
              undefined,
            ]
          }
        >
          <View
            style={
              {
                "flex": 1,
                "position": "relative",
              }
            }
          >
            <View
              accessibilityIgnoresInvertColors={true}
              style={
                {
                  "flex": 1,
                  "justifyContent": "center",
                }
              }
            >
              <Image
                blurRadius={70}
                resizeMode="cover"
                source={1}
                style={
                  [
                    {
                      "bottom": 0,
                      "left": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    },
                    {
                      "height": undefined,
                      "width": undefined,
                    },
                    undefined,
                  ]
                }
              />
              <View
                style={
                  {
                    "alignItems": "center",
                    "flex": 1,
                    "flexDirection": "row",
                    "justifyContent": "center",
                  }
                }
              >
                <View
                  collapsable={false}
                  style={
                    {
                      "backgroundColor": "transparent",
                      "overflow": "hidden",
                      "transform": [
                        {
                          "rotate": "0deg",
                        },
                      ],
                    }
                  }
                  useNativeDriver={false}
                >
                  <RNSVGSvgView
                    bbHeight="140"
                    bbWidth="140"
                    focusable={false}
                    height={140}
                    style={
                      [
                        {
                          "backgroundColor": "transparent",
                          "borderWidth": 0,
                        },
                        {
                          "flex": 0,
                          "height": 140,
                          "width": 140,
                        },
                      ]
                    }
                    width={140}
                  >
                    <RNSVGGroup
                      fill={
                        {
                          "payload": 4278190080,
                          "type": 0,
                        }
                      }
                    >
                      <RNSVGPath
                        d="M64.50916208442445 15.274770909708586
          A55 55 0 0 1 70 15"
                        fill={
                          {
                            "payload": 0,
                            "type": 0,
                          }
                        }
                        propList={
                          [
                            "fill",
                            "stroke",
                            "strokeWidth",
                            "strokeLinecap",
                          ]
                        }
                        stroke={
                          {
                            "payload": 4278956978,
                            "type": 0,
                          }
                        }
                        strokeLinecap={1}
                        strokeWidth="10"
                      />
                    </RNSVGGroup>
                  </RNSVGSvgView>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
      <RNSScreenStackHeaderConfig
        backButtonInCustomView={false}
        backTitleVisible={true}
        backgroundColor="rgb(255, 255, 255)"
        color="rgb(0, 122, 255)"
        direction="ltr"
        disableBackButtonMenu={false}
        hidden={true}
        hideBackButton={false}
        largeTitleHideShadow={false}
        title="Weather"
        titleColor="rgb(28, 28, 30)"
        topInsetEnabled={false}
        translucent={false}
      />
    </RNSScreen>
  </RNSScreenStack>
</RNCSafeAreaProvider>
`;
